### PR TITLE
Add summary and description to openapi

### PR DESCRIPTION
**Pull request by achamayou**
Addresses some of #1920. Common endpoints and governance are covered, but I will hold off writing all the descriptions until we've completed our API review in the context of #4469.

I would like to merge this in the meantime.

### CODE REVIEW

File: tests/infra/network.py [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ def wait_for_all_nodes_to_commit(self, primary=None, tx_id=None, timeout=10):
                 with node.client() as c:
                     logs[node.node_id] = []
                     resp = c.get(
-                        f"/node/local_tx?transaction_id={tx_id}",
+                        f"/node/tx?transaction_id={tx_id}",
```
Comment (eddyashton) [inline tests/infra/network.py:1050]: Once upon a time, `local_tx` was never forwarded, and that was necessary for some tests to pass. Looks like that's no longer the case though, so if the CI is happy let's purge it.

**Review by eddyashton — APPROVED** [review]
LGTM!

File: tests/infra/network.py (hunk 1 above)
Comment (achamayou) [reply tests/infra/network.py:1050]: Yes, I think it ceased being useful when "set_execute_outside_consensus" was removed.

File: src/endpoints/common_endpoint_registry.cpp [hunk 2]
```diff
@@ -237,15 +240,10 @@ namespace ccf
...
-      "/local_tx",
-      HTTP_GET,
-      json_command_adapter(get_tx_status),
-      no_auth_required)
-      .set_auto_schema<void, GetTxStatus::Out>()
-      .add_query_parameter<ccf::TxID>(tx_id_param_key)
+      .set_openapi_summary("Current status of a transaction")
+      .set_openapi_description(
+        "Possible statuses returned are Unknown, Pending, Committed or "
+        "Invalid.")
```
[the remark this answers is not in the document]
Comment (achamayou) [reply src/endpoints/common_endpoint_registry.cpp:246]: I thought so too at first, but our OpenAPI generation isn't that good, and so it's not there: https://microsoft.github.io/CCF/main/use_apps/rpc_api.html#get--app-tx

I don't think it's likely the statuses will change any time soon, or any sooner than the move to CADL anyway, so I claim this is fine for now.

**Pull request merged**